Update README

yash-puligundla opened the pull request:
- update readme
- add parameterLabel field to username and password for create-tables and add-test-dataset commands

jb-adams committed:
Merge pull request #4 from ga4gh/release/0.1.1

Release 0.1.1 into main

yash-puligundla committed:
Merge pull request #9 from ga4gh/release/0.1.2

Release/0.1.2

Commit:
add parameterLabel fields to username and password for create-tables and add-test-dataset subcommands

Commit:
Update readme

emre-f commented:
Here are my comments:

1. This PR is into main, is that intended? Shouldn't it be into develop? 
2. In the first native running part of `README.md`, you have `starter-kit-data-connect` there in the command. Is that intended? If not there are many instances of that command that should be changed, but I am not sure if `starter-kit-utils` uses `data-connect` database.
3. In the `./gradlew bootJar` code block there is an extra space.
4. The title "When source is a url", url should probably be capitalized to URL. (Same in the Docker part)
5. Under `list-migrations` title there is an "Example" row that is completely empty
6. There are empty cells in under the `create-tables` and `add-test-dataset` tables, are those intended? 
7. Similarly, for the two tables mentioned above I believe it would look cleaner if the entries in the "Parameter Name" column are separated into two separate lines. Right now the second parameter names get cut in half.

Commit:
clean up

yash-puligundla commented:
Thanks for the comments. I am adding the replies inline

> Here are my comments:
> 
> 1. This PR is into main, is that intended? Shouldn't it be into develop?
**Good catch! Changed it to develop**

> 2. In the first native running part of `README.md`, you have `starter-kit-data-connect` there in the command. Is that intended? If not there are many instances of that command that should be changed, but I am not sure if `starter-kit-utils` uses `data-connect` database.
**This is intentional as starter-kit-utils can be used to create databases for ga4gh-starter-kits. I have used data-connect as an example.**

> 3. In the `./gradlew bootJar` code block there is an extra space.
**removed the extra space**

> 4. The title "When source is a url", url should probably be capitalized to URL. (Same in the Docker part)
**Done**

> 5. Under `list-migrations` title there is an "Example" row that is completely empty
**changed it**

> 6. There are empty cells in under the `create-tables` and `add-test-dataset` tables, are those intended?
**yes, this is intentional. I left them blank if there is no associated values**

> 7. Similarly, for the two tables mentioned above I believe it would look cleaner if the entries in the "Parameter Name" column are separated into two separate lines. Right now the second parameter names get cut in half.
**I agree, but, unfortunately, I couldn't find a way to make the "Parameter Name" values look elegant. If I added a line break, it messes up the table.**

emre-f reviewed (approved):
Looks good to commit now.
